internal/service/service.go

```diff
@@ -20,10 +19,25 @@ func NewServ(repository Repository) *Serv {
 	}
 }
 
-func (t Serv) CreatePosition(p *internal.Position) error {
+func logCorrelationID(ctx context.Context) error {
+	corelationID, ok := ctx.Value("ID").(string)
+	if !ok {
+		return errors.StatusInternalServerError()
+	}
+	logrus.WithFields(logrus.Fields{
+		"corelation_id": corelationID,
```

Comment:
Please use the same key in context and in logs

Reply:
+